test/train-sets/ref/ccb_reuse_small.stderr

```diff
@@ -8,13 +8,13 @@ num sources = 1
 Enabled reductions: gd, scorer, csoaa_ldf, cb_adf, cb_explore_adf_greedy, cb_sample, shared_feature_merger, ccb_explore_adf
 average  since         example        example  current  current  current
 loss     last          counter         weight    label  predict features
-0.000000 0.000000            1            1.0 0:0.000000        0        8
-0.000000 0.000000            2            2.0 1:0.000000        1        8
-0.000000 0.000000            4            4.0 1:0.000000        1        8
+0.000000 0.000000            1            1.0 0:0.000000        0       12
+0.000000 0.000000            2            2.0 1:0.000000        1       12
+0.000000 0.000000            4            4.0 1:0.000000        1       12
 
 finished run
 number of examples = 4
 weighted example sum = 4.000000
 weighted label sum = 0.000000
 average loss = 0.000000
-total feature number = 32
+total feature number = 48
```

Comment:
what is the explanation behind these 2 counters, for example how does one arrive at 8 or 12 features in this one example?

ccb shared |User b
ccb action |Action d
ccb action |Action e
ccb slot 0:0:0.2 |Slot h